### PR TITLE
Short urls

### PR DESCRIPTION
**Pull request by arcturus**

### CODE REVIEW

**Comment by arcturus** [comment]
@wilsonpage reviews?

This will fix #24 and #23

**Comment by arcturus** [comment]
@wilsonpage i took the freedom to deploy it on tegam .... what could go wrong?

**Comment by arcturus** [comment]
I think it solves #21 too.

**Comment by arcturus** [comment]
Went for this decision since some webviews could have problems with urls of more than 2000 characters.

File: views/embed.jade [hunk 1]
```diff
@@ -4,60 +4,12 @@ block content
   script.
     document.addEventListener('DOMContentLoaded', function() {
       var events=!{JSON.stringify(events)};
```
Comment (wilsonpage) [inline views/embed.jade:6]: Redefinition of `event`. Should this be:

``` js
var events=!{JSON.stringify(window.events)};
```

?

**Comment by wilsonpage** [comment]
My advice would be:
- Remove oembed logic for this app completely and use oembed-adaptor
- Have just one route `/?url=` that:
  1. Scrapes sched
  2. Populates `window.events` JSON
  3. Leaves client-side to render

This would:
- Remove a lot of complexity
- Removes risk of hitting URL length limit
- Use local client-side time to avoid TZ issues

But maybe this is too much work.

File: views/embed.jade (hunk 1 above)
Comment (wilsonpage) [inline views/embed.jade:6]: Comment here to explain what is Jade templating syntax and what is JS.

**Comment by wilsonpage** [comment]
Fine for this to ship considering the time restrictions.

**Comment by wilsonpage** [comment]
But a comment to explain the funky parts.

**Comment by arcturus** [comment]
Fair enough

**Comment by arcturus** [comment]
I think you are right on the comment, but as you acknowledge, for timing restrictions we will patch the current solution.

**Pull request merged**